cli_reference/index.adoc

```diff
@@ -0,0 +1,38 @@
+[id="cli-tools-overview"]
+= {product-title} CLI tools overview
+include::modules/common-attributes.adoc[]
+:context: cli-tools-overview
+
+toc::[]
+
+A user performs different tasks while working on {product-title} which includes a range of operations such as the following:
+
+* Managing clusters
+* Building, deploying, and managing applications
+* Managing deployment processes
+* Developing Operators
+* Creating and maintaining Operator catalogs
+
+{product-title} offers a set of command-line interface (CLI) tools that simplify these tasks by enabling users to perform various administration and development operations from the terminal.
+These tools expose simple commands to manage the applications, as well as the lower level tools to interact with each component of the system.
+
+[id="cli-tools-list"]
+== List of CLI tools
+//Some of these CLI tools are alternatively available in web console
+
+// Provide Getting started xref for all the CLI tools
+The following set of CLI tools are available in {product-title}:
+
+* xref:../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[OpenShift CLI (oc)]: This is the most commonly used CLI tool by {product-title} users. It helps both cluster administrators and developers to perform end-to-end operations across {product-title} using the terminal. Unlike the web console, it allows the user to work directly with the project source code using command scripts.
+
+* xref:../cli_reference/developer_cli_odo/understanding-odo.adoc#understanding-odo[Developer CLI (odo)] (not in 4.8 and 4.9): The developers, who primarily create and maintain applications on {product-title}, do not require a detailed knowledge of {product-title}. `odo` helps the developers to to write, build, and debug applications on a cluster from the terminal without the need to have an understanding of cluster administration.
```

Comment:
Suggest a bit of rewording: `odo` helps developers focus on their main goal of creating and maintaining applications on {product-title} by abstracting away complex Kubernetes and OpenShift Container Platform concepts.  It helps the developers to write, build, and debug applications on a cluster from the terminal without the need to administer the cluster.

Reply:
Since it is already in 4.8, suggest removing it only for 4.9, but let me confirm this.